package.json

```diff
@@ -64,7 +64,7 @@
     "eslint-plugin-promise": "6.0.0",
     "eslint-plugin-standard": "5.0.0",
     "faker": "5.5.3",
-    "globby": "11.1.0",
+    "globby": "13.1.1",
```

Comment:
globby [v12 breaking changes](https://github.com/sindresorhus/globby/releases/tag/v12.0.0) include using named exports instead of a default export which would break all of the existing usages of the package. Also [globby v13 breaking changes](https://github.com/sindresorhus/globby/releases/tag/v13.0.0) renames and removes some of the old API interface. So I from what I've seen from v12 and v13 releases. I don't think it's worth spending anytime in migrating to v13, it now works and nothing revolutionary happened related to the way we're using it. so I'll close it for now but feel free to share your thoughts and/or re-open it 🤗